core/src/main/resources/defaults/blueprints/cdp-sdx-dev.bp

```diff
@@ -0,0 +1,288 @@
+                        "refName": "hbase-HBASETHRIFTSERVER-BASE",
+                        "roleType": "HBASETHRIFTSERVER"
+                    }
+                ],
+                "serviceType": "HBASE"
+            },
+            {
+                "refName": "hdfs",
+                "roleConfigGroups": [
+                    {
+                        "base": true,
+                        "refName": "hdfs-NAMENODE-BASE",
+                        "roleType": "NAMENODE"
+                    },
+                    {
+                        "base": true,
+                        "refName": "hdfs-SECONDARYNAMENODE-BASE",
+                        "roleType": "SECONDARYNAMENODE"
+                    },
+                    {
+                        "base": true,
+                        "refName": "hdfs-DATANODE-BASE",
+                        "roleType": "DATANODE"
+                    },
+                    {
+                        "base": true,
+                        "refName": "hdfs-BALANCER-BASE",
+                        "roleType": "BALANCER"
+                    },
+                    {
+                        "base": true,
+                        "refName": "hdfs-GATEWAY-BASE",
+                        "roleType": "GATEWAY"
+                    },
+                    {
+                        "base": true,
+                        "refName": "hdfs-NFSGATEWAY-BASE",
+                        "roleType": "NFSGATEWAY"
+                    }
+                ],
+                "serviceType": "HDFS"
+            },
+            {
+                "refName": "kafka",
+                "roleConfigGroups": [
+                    {
+                        "base": true,
+                        "refName": "kafka-GATEWAY-BASE",
+                        "roleType": "GATEWAY"
+                    },
+                    {
+                        "base": true,
+                        "refName": "kafka-KAFKA_BROKER-BASE",
+                        "roleType": "KAFKA_BROKER"
+                    }
+                ],
+                "serviceConfigs": [
+                    {
+                        "name": "zookeeper_service",
+                        "ref": "zookeeper"
+                    },
+                    {
+                        "name": "offsets.topic.replication.factor",
+                        "value": "1"
+                    }
+                ],
+                "serviceType": "KAFKA"
+            },
+            {
+                "serviceType": "KNOX",
+                "refName": "knox",
+                "roleConfigGroups": [
+                    {
+                        "base": true,
+                        "refName": "knox-KNOX_GATEWAY-BASE",
+                        "roleType": "KNOX_GATEWAY",
+                        "configs": [
+                            {
+                                "name": "gateway_master_secret",
+                                "value": "cloudera1"
```

Comment:
I think it would be better to use the cluster "master" password instead, which can be referenced as:

https://github.com/hortonworks/cloudbreak/blob/15ae3258db8dd505f0bafcf63985bb882ee77b61/core/src/main/resources/defaults/blueprints/cdp-sdx.bp#L81-L82

CB requires minimum 8-chars with at least one number for the cluster password.  I didn't find requirements for Knox Gateway master secret, but based on the example I assume it would accept the same.